bundle-workflow/src/test_workflow/test_cluster.py

```diff
@@ -22,7 +22,7 @@ def create(cls, *args):
         cluster = cls(*args)
         try:
             cluster.create_cluster()
-            yield cluster.endpoint()
+            yield cluster.endpoint(), cluster.port()
```

Comment:
You could just `yield cluster` here and it would have `endpoint` and `port` properties, but this is fine as long as it's no more than a tuple.